tests/openwisp2/settings.py

```diff
@@ -170,6 +176,8 @@
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 CELERY_EMAIL_BACKEND = EMAIL_BACKEND
 
+test_query = importlib.import_module(TIMESERIES_DATABASE["BACKEND"]).queries.test_query
```

Comment:
I don't think we should be doing this.
In this section, some custom queries used in the tests are defined (we'll have to change this code to use register()), I think the definition of the test queries must stay here.